File: 3º ano/1º Semestre/DSS/Trabalho/Fase3/MediaCenter/src/Business/Gestor_Contas/Utilizador.java
package Business.Gestor_Contas;

public class Utilizador extends Utente{
    public Utilizador(int id, String nome, String email, String password) {
        super(id, nome, email, password);
    }

    public Utilizador(Utente ut) {
        super(ut);
    }

    public Utilizador() { super(); }
}

File: 3º ano/2º Semestre/LI4/Trabalho final/BackofficeSOSA/BackofficeSOSA/BackofficeSOSA/Models/Utilizador.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace BackofficeSOSA.Models
{
    public class Utilizador
    {
        
        public int Id { get; set; }

        [Required]
        [DataType(DataType.Text)]
        public string Nome { get; set; }

        [Required]
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }

        [Required]
        [DataType(DataType.Password)]
        public string Password { get; set; }

        [Required]
        [DataType(DataType.PhoneNumber)]
        public int NrTelemovel { get; set; }
    }
}